app/src/main/java/fr/free/nrw/commons/media/MediaDetailFragment.java

```diff
@@ -9,11 +9,14 @@
 import android.os.Bundle;
 import android.text.Editable;
 import android.text.TextWatcher;
+import android.util.DisplayMetrics;
 import android.util.TypedValue;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.view.ViewTreeObserver;
+import android.view.ViewTreeObserver.OnGlobalLayoutListener;
```

Comment:
This should be the only new import I think

Reply:
yes indeed

Reply:
no it shows error otherwise, both imports would be required

Reply:
![image](https://user-images.githubusercontent.com/3358282/78673314-dbdab000-78d9-11ea-8cef-d2e961133784.png)

The other imports, press `ctrl+alt+o` and turn these options on

![image](https://user-images.githubusercontent.com/3358282/78673435-09bff480-78da-11ea-8761-b28f23f051b3.png)

Reply:
Oh I'll fix those asap